src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$font-color: #292929;
+$font-color-form: #7e7e83;
+$pargraph-font-family: "Space Mono", sans-serif;
+$input-font-family: "Space Mono", sans-serif;
```

Comment:
You are using the same font for both input and paragraph. And same for link and title, remove duplicated variables and change the name to be more generic like e.g. `$primary-font-family` and `$secondary-font-family`